include/modem/location.h

```diff
@@ -250,6 +258,12 @@ struct location_config {
 	 * the valid range is 10...65535 seconds.
 	 */
 	uint16_t interval;
+
+	/**
+	 * @brief Location acquisition mode.
+	 *
```

Comment:
The empty line can be removed if there's no detailed description.

Reply:
removed